app/scripts/update-translation.js

```diff
@@ -0,0 +1,98 @@
+import fs from 'node:fs/promises'
+import path from 'node:path'
+import { fileURLToPath } from 'url';
+import { dirname } from 'path';
+import { generateText } from 'ai';
+import { openai } from '@ai-sdk/openai';
+
+// Convert the module's URL to a file path
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+
+if (!process.env.OPENAI_API_KEY) {
+  console.error('Please set the OPENAI_API_KEY environment variable')
+  process.exit(1)
+}
+
+const model = openai('gpt-4');
```

Comment:
I would use 'gpt-4o-mini' here over 'gpt-4'. It is much faster and cheaper and for simple translation tasks sufficient.

Reply:
OK!  Since I already ran everything with gpt-4, I added a task to change the model later. 
https://openearth.atlassian.net/browse/ON-3127